pkg/scheduler/cache/cache_mock.go

```diff
@@ -0,0 +1,127 @@
+package cache
+
+import (
+	"math"
+	"os"
+	"strconv"
+
+	schedulingv1 "k8s.io/api/scheduling/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/workqueue"
+
+	vcclient "volcano.sh/apis/pkg/client/clientset/versioned/fake"
+	"volcano.sh/volcano/cmd/scheduler/app/options"
+	schedulingapi "volcano.sh/volcano/pkg/scheduler/api"
+)
+
+// NewCustomMockSchedulerCache returns a mock scheduler cache with custom interface
+func NewCustomMockSchedulerCache(schedulerName string,
```

Comment:
Who will call this?

Reply:
Anyone want to use it . Eg, UT packages.